metaflow/plugins/argo/argo_workflows_decorator.py

```diff
@@ -113,8 +113,15 @@ def task_finished(
         # finished execution.
 
         if self.attributes["auto-emit-argo-events"]:
-            # Event name is set to metaflow.flow.step
+            # Event name is set to metaflow.project.branch.step so that users can
+            # place explicit dependencies on namespaced events. Also, argo events
+            # sensors don't allow for filtering against absent fields - which limits
+            # our ability to subset non-project namespaced events.
             # TODO: Check length limits for fields in Argo Events
+            event = ArgoEvent(
+                name="metaflow.%s.%s"
+                % (current.get("project_flow_name", flow.name), step_name)
```

Comment:
something is not working with this part:
- during task execution, `current.project_flow_name` returns correctly `sa_test.user.saikonen.EventTriggeringTest` for my test
- the event that is auto-emitted is `metaflow.EventTriggeringTest.start` though.